pkg/proxy/backup.go

```diff
@@ -69,7 +74,72 @@ func (p *Proxy) SnapshotBackupStatus(ctx context.Context, req *rpc.EngineSnapsho
 
 	resp = &rpc.EngineSnapshotBackupStatusProxyResponse{}
```

Comment:
NIT: same here. don't need to init the response at the beginning.

Reply:
Updated.

Reply:
Updated.